CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ See also [TiDB change log][tidb_change_log] and [PD change log][pd_change_log].
 [tidb_change_log]: https://github.com/pingcap/tidb/blob/master/CHANGELOG.md
 [pd_change_log]: https://github.com/pingcap/pd/blob/master/CHANGELOG.md
 
+## [2.0.1] - 2018-05-10
```

Comment:
2018-05-11?

Reply:
Actually the day is not settle down yet, I'm going to remove it.